src/flexible_alert.c

```diff
@@ -265,6 +265,20 @@ flexible_alert_handle_metric (flexible_alert_t *self, fty_proto_t **ftymsg_p)
     const char *quantity = fty_proto_type (ftymsg);
```

Comment:
Should we `assert` that `quantity != NULL`, given dereferences below?

Reply:
Perhaps so, but this part of the code would require some refactoring. Either this or the one in fty-alert-engine (and minor change here).